vocab-builder-core/src/main/java/com/github/tkurz/sesame/vocab/GenerationSetting.java

```diff
@@ -0,0 +1,53 @@
+package com.github.tkurz.sesame.vocab;
+
+import com.google.common.base.CaseFormat;
+
+public class GenerationSetting {
+
+	private String caseFormat;
+	private String constantPrefix;
+	private String constantSuffix;
+	public CaseFormat getCaseFormat() {
+		try {
+			if ( caseFormat != null ) {
```

Comment:
No need for this null check if we are catching all exceptions. CaseFormat.valueOf will throw an NPE for that.

Alternatively, the null check could be outside of the try-catch block to emphasise that the exceptions are expected from CaseFormat.valueOf